### PR TITLE
server: Add new privacy revision revision-20200205.twig

### PR DESCRIPTION
**Pull request by kulla**
Follow up of #102 (Since there is a bigger time delay I created a new PR). Here are the exact changes made:

```
 ▶ diff revision-20181201.twig revision-20200205.twig                                                            
32,35c32,34
<                 und Weise der Nutzung von Serlo.org durch unsere BesucherInnen analysieren,
<             </li>
<             <li>um Feedback zu unseren Lehrinhalten zu erhalten und diese verbessern und weiterentwickeln zu können
<             </li>
---
>                 und Weise der Nutzung von Serlo.org durch unsere BesucherInnen analysieren,</li>
>             <li>um Feedback zu unseren Lehrinhalten zu erhalten und diese verbessern und weiterentwickeln zu können
>             <li>um die Arbeit in unserer Community zu organisieren und in Kontakt mit Autor*innen treten.</li>
273a273,281
>         <h4>Aktivitätsdaten</h4>
>         <p>Wenn du auf Serlo.org registriert und als Autor*in oder Reviewer*in aktiv bist, wird deine Bearbeitungsh
>             in unserer Datenbank gespeichert. Serlo erfasst jeweils deinen Nutzernamen und Zeitpunkt der Aktivität.
>             Auswertung dieser Daten ermöglicht es Serlo , die Begleitung unserer Onlinecommunity zu verbessern und 
>             zu stellen, dass auf jeden Beitrag eingegangen werden kann. Die Auswertung der Daten erfolgt über Monit
>             und Analyseprogrammen.</p>
>         <p>Die Datenverarbeitung dient der Verbesserung der Wirkungsweise der Plattform. Die Datenverarbeitung gesc
>             auf Grundlage des Art. 6 Abs. 1 lit. f DSGVO. Der Webseitenbetreiber hat ein berechtigtes Interesse an 
>             Auswertung der Daten zur Verbesserung des Online-Angebots.</p>
```

### CODE REVIEW

File: packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20200205.twig [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+    <div class="c24">
+        <h2>Warum erfassen wir personenbezogene Daten</h2>
+        <p>Serlo.org nutzt personenbezogene Daten zu folgenden Zwecken:</p>
+        <ul>
+            <li>um die Dienste von Serlo anzubieten und verbessern zu können,</li>
+            <li>um zu verstehen, wie BesucherInnen Serlo.org nutzen; dadurch können wir sicherstellen, dass Serlo.org
+                ordnungsgemäß funktioniert, wir können neue Nutzungsmöglichkeiten und Features entwickeln und die Art
+                und Weise der Nutzung von Serlo.org durch unsere BesucherInnen analysieren,</li>
+            <li>um Feedback zu unseren Lehrinhalten zu erhalten und diese verbessern und weiterentwickeln zu können.</li>
+            <li>um die Arbeit in unserer Community zu organisieren und in Kontakt mit Autor*innen treten.</li>
```
Comment (kulla) [inline packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20200205.twig]: This line has changed...

File: packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20200205.twig [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+            registriert sind und werden anschließend gelöscht. Gesetzliche Aufbewahrungsfristen bleiben unberührt.</p>
+        <h4>Aktivitätsdaten</h4>
+        <p>Wenn du auf Serlo.org registriert und als Autor*in oder Reviewer*in aktiv bist, wird deine Bearbeitungshistorie
+            in unserer Datenbank gespeichert. Serlo erfasst jeweils deinen Nutzernamen und Zeitpunkt der Aktivität. Die
+            Auswertung dieser Daten ermöglicht es Serlo , die Begleitung unserer Onlinecommunity zu verbessern und sicher
+            zu stellen, dass auf jeden Beitrag eingegangen werden kann. Die Auswertung der Daten erfolgt über Monitoring- 
+            und Analyseprogrammen.</p>
+        <p>Die Datenverarbeitung dient der Verbesserung der Wirkungsweise der Plattform. Die Datenverarbeitung geschieht
+            auf Grundlage des Art. 6 Abs. 1 lit. f DSGVO. Der Webseitenbetreiber hat ein berechtigtes Interesse an der
+            Auswertung der Daten zur Verbesserung des Online-Angebots.</p>
```
Comment (kulla) [inline packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20200205.twig]: Those section was added...

File: packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20200205.twig [hunk 3]
```diff
@@ -0,0 +1,517 @@
...
+        <p>Die Verarbeitung der bei Registrierung angegebenen Daten geschieht auf Grundlage des Art. 6 Abs. 1 lit. a
+            DSGVO mit Deiner Einwilligung. Du kannst eine von Dir erteilte Einwilligung jederzeit widerrufen. Dazu
+            reicht eine formlose Mitteilung per E-Mail an uns. Die Rechtmäßigkeit der bereits erfolgten
+            Datenverarbeitung bleibt vom Widerruf unberührt.</p>
+        <p>Die bei der Registrierung erfassten Daten werden von uns gespeichert, solange Du auf unserer Website
+            registriert sind und werden anschließend gelöscht. Gesetzliche Aufbewahrungsfristen bleiben unberührt.</p>
+        <h4>Aktivitätsdaten</h4>
+        <p>Wenn du auf Serlo.org registriert und als Autor*in oder Reviewer*in aktiv bist, wird deine Bearbeitungshistorie
+            in unserer Datenbank gespeichert. Serlo erfasst jeweils deinen Nutzernamen und Zeitpunkt der Aktivität. Die
+            Auswertung dieser Daten ermöglicht es Serlo , die Begleitung unserer Onlinecommunity zu verbessern und sicher
```
Comment (inyono) [inline packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20200205.twig]:
```suggestion
            Auswertung dieser Daten ermöglicht es Serlo, die Begleitung unserer Onlinecommunity zu verbessern und sicher
```

Comment (inyono) [inline packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20200205.twig]:
```suggestion
            Auswertung dieser Daten ermöglicht es Serlo, die Begleitung unserer Online-Community zu verbessern und sicher
```

File: packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20200205.twig [hunk 4]
```diff
@@ -0,0 +1,517 @@
...
+            zu stellen, dass auf jeden Beitrag eingegangen werden kann. Die Auswertung der Daten erfolgt über Monitoring- 
+            und Analyseprogrammen.</p>
```
Comment (inyono) [inline packages/public/server/src/module/Ui/templates/static/de/datenschutz/revision-20200205.twig]:
```suggestion
            und Analyseprogramme.</p>
```

**Comment by kulla** [comment]
> Two minor changes

Thanks! :-) I have added them.

**Pull request merged**